spp_change_request/views/dms_file_view.xml

```diff
@@ -27,7 +27,6 @@
                                 options="{'no_open': True,'no_create': True,'no_edit':True}"
                                 required="1"
                                 readonly="context.get('category_readonly')"
-                                domain="[('parent_id', 'child_of', %(dms.category_01_demo)d)]"
```

Comment:
@gonzalesedwin1123 pls don't remove this line. It is up to the request from the issue 186858

Reply:
OK @nhatnm0612, thanks for the info but I will just add a category for this. It is generating an error in the installation  of the spp_change_request if the "Install demo data" is off.

Reply:
@gonzalesedwin1123 is the category added? Can we merge this?

Reply:
@jeremi the category is now in the latest commit. It can be merged now.